### PR TITLE
chore: Remove bolding from headings

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Headings already have bold font weighing, so the extra `<strong>` tags aren't needed

### What's being changed:

Remove `**` from headings so extra `<strong>` tags aren't added

### Check off the following:
- [x] All of the tests are passing.
- [ ] I have reviewed my changes in staging.
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/actions/creating-actions/metadata-syntax-for-github-actions.md [hunk 1]
```diff
@@ -22,19 +22,19 @@ Docker and JavaScript actions require a metadata file. The metadata filename mus
 
 Action metadata files use YAML syntax. If you're new to YAML, you can read "[Learn YAML in five minutes](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes)."
 
-### **`name`**
+### `name`
```
Comment (nschonni) [inline content/actions/creating-actions/metadata-syntax-for-github-actions.md:25]: This actually does come out a little lighter, but I didn't dig into the CSS on why

**Comment by janiceilene** [comment]
Thanks @nschonni! I'll triage this for review ✨

**Comment by chiedo** [comment]
@janiceilene this is content! Can you assign to the Docs Content team?

**Comment by janiceilene** [comment]
🙈 Thanks for catching that @chiedo!

**Review by shati-patel — APPROVED** [review]
Thank you for tidying up all these headings, @nschonni! 🎉 I agree, we don't need them to be bold 👍🏽 

I'll get this merged shortly!

File: content/actions/creating-actions/metadata-syntax-for-github-actions.md (hunk 1 above)
Comment (shati-patel) [reply content/actions/creating-actions/metadata-syntax-for-github-actions.md:25]: You're right, there's a slight visual difference, but that seems fine 🙂

**Pull request merged**